nomic/atlas.py

```diff
@@ -216,16 +364,15 @@ def map_text(
     first_sample = next(data_iterator, None)
 
     if first_sample is None:
-        logger.warning("Passed data has no samples. No project will be created")
+        logger.warning("Passed data has no samples. No dataset will be created")
         return
 
-    project = AtlasProject(
-        name=project_name,
+    project = AtlasDataset(
+        identifier=project_name,
         description=description,
         unique_id_field=id_field,
         modality='text',
```

Comment:
Can we make the necessary atlas_cloud changes to drop modality from the AtlasProject arguments? Even if we retain projects as having necessary modalities for the time being, we can defer deciding what the type of a project is until the user calls `upload_text` or `upload_embeddings`. I think the plan I wrote [here](https://www.notion.so/nomic-ai/Multiple-Modalities-in-Atlas-9cb602a28e2d47d9b67a6adf12379035) is still workable.

Reply:
done